fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -5637,5 +5640,59 @@ public LogicalPlan visitShowQueuedAnalyzeJobs(ShowQueuedAnalyzeJobsContext ctx)
         String stateValue = ctx.stateValue == null ? null : stripQuotes(ctx.stateValue.getText());
         return new ShowQueuedAnalyzeJobsCommand(tableName, stateKey, stateValue);
     }
+
+    @Override
+    public LogicalPlan visitDescribeTable(DorisParser.DescribeTableContext ctx) {
+        String tableName = null;
+        if (ctx.multipartIdentifier() != null) {
```

Comment:
explainCommand multipartIdentifier specifiedPartition?    #describeTable

multipartIdentifier is always not null and specifiedPartition is missing